licenses/LICENSE-opensearch

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2023 Canonical Ltd.
```

Comment:
I think we cannot put Canonical copyrights into opensearch license, moreveover they didn't fill it well: https://github.com/opensearch-project/OpenSearch/blob/main/LICENSE.txt#L189

@Mehdi-Bendriss should we report this to opensearch as an issue? :-)